airbyte-webapp/src/components/EntityTable/components/FrequencyCell.tsx

```diff
@@ -27,7 +27,7 @@ const FrequencyCell: React.FC<FrequencyCellProps> = ({ value, enabled, scheduleT
     <Content enabled={enabled}>
       <FormattedMessage
         id={`frequency.${value ? value.basicSchedule?.timeUnit : "manual"}`}
```

Comment:
```suggestion
        id={`frequency.${value?.basicSchedule?.timeUnit ?? "manual"}`}
```
more consistent since we're optional chaining below